plan/config.go

```diff
@@ -29,13 +29,17 @@ import (
 	"time"
 )
 
-const defaultCallTimeout = 5
+const (
+	defaultCallTimeout = 5
+	defaultHostTimeout = 30
```

Comment:
nit - if you're not using typed values, then indicate the units in the the var name, defaultCallTimeoutInSeconds